Sources/ArgumentParser.swift

```diff
@@ -239,4 +239,26 @@ public final class ArgumentParser : ArgumentConvertible, CustomStringConvertible
 
     return nil
   }
+  
+  /// Returns the value for an option (--name Kyle, --name=Kyle) or flag (-n Kyle)
+  public func shiftValueForOption(_ name: String, orFlag flag: Character?) throws -> String? {
```

Comment:
I'm not sure if I like this function name (with `orFlag`), although I don't have any better ideas at the moment.

Gonna have a think about this.